### DISCUSSION TITLE
请问streampark支持HA高可用吗

### DISCUSSION THREAD

**Discussion by labixiaoxiaopang** [apache/streampark#2745] [Q&A] [answered]
在两台节点上同时部署streampark，共用一个MySQL数据源，是否可以实现高可用，有没有并发问题？

**Comment by caicancai** [comment, marked as answer]
It is not supported at present, and it is estimated that this part will be designed in the future

**Comment by Al-assad** [comment]
StreamPark is just a lightweight monolithic application. There are many concurrency issues when multiple SP instances share persistent storage, such as conflicts between local workspace files on multiple nodes and conflicts in RDBMS data. 
Of course, the HA of SP is currently within the development plan.😃
